### DISCUSSION TITLE
Sonic's Moveset

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#138] [Q&A]
What is Sonic's moveset in Sonic Worlds Next?

**Comment by DimensionWarped** [comment]
Why not just play it and find out? We don't really chart things in terms of movesets. Everyone gets a spindash and a second jump action (sometimes a variable second jump action). The main thing that sets Sonic apart are Sonic 3 style shield actions (instant + elements) and drop dash (both of which are separate second jump actions, but drop dash is a little unusual), and I guess the one thing that really makes Sonic stand out from the rest since no other characters map an ability this way is the peel-out.

Never really cared much for the peel-out personally... always felt like it was more than a little redundant, but it's there for the Sonic CD fans.
